build.sbt

```diff
@@ -4834,13 +4835,18 @@ lazy val `std-table` = project
     },
     libraryDependencies ++= Seq(
       "org.graalvm.polyglot"     % "polyglot"                % graalMavenPackagesVersion % "provided",
+      "org.graalvm.truffle"      % "truffle-api"             % graalMavenPackagesVersion,
```

Comment:
You probably want either a `% "provided"` or `% "Test"` classifier.

Reply:
This might be the source of my license check problems :)